src/pages/ProductList/ProductList.js

```diff
@@ -1,7 +1,161 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import Nav from './../../components/Nav/Nav';
+import Category from './Category/Category';
+import FilterOpen from './FilterOpen/FilterOpen';
+import BaseInfo from '../../components/BaseInfo/BaseInfo';
+import Footer from './../../components/Footer/Footer';
+import './ProductList.scss';
 
 const ProductList = () => {
-  return <div>wesop</div>;
+  const [productList, setProductList] = useState([]);
+  const [isFilterOpen, setIsFilterOpen] = useState(false);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/category_products.json')
+      .then(res => res.json())
+      .then(data => setProductList(data));
+  }, []);
+
+  const filterClickHandler = () => {
+    setIsFilterOpen(!isFilterOpen);
+  };
+
+  return (
+    <div className="ProductList">
+      <Nav />
+      <header>
+        <Link to="/">
+          <img className="logo" alt="logo" src="/images/common/Wesop.png" />
+        </Link>
+        <h1 className="mainCategory">스킨</h1>
+      </header>
+      <div className="filterWrapper">
+        <ul className="filterSubNavContainer">
+          <li className="filterSubNavList">
+            <button className="filterSubNavLink">모든 스킨 </button>
+          </li>
+          {CATEGORY_LIST.map(category => {
```

Comment:
```suggestion
          {CATEGORY_LIST.map(({id,category}) => {
```

* map도 매개변수 자리에서 바로 구조분해할당 할 수 있습니다

Reply:
다음과 같이 보내고 받는 것이 맞을까요? 작동은 되는데 기존에 `{...category}`와 같은 형태로 통째로 보내는 것이 더 간결해 보여서 여쭙습니다 😀

보낼 때
```{productList.map(
          ({ categoryId, categoryName, categoryDescription, products }) => {
            return (
              <Category
                key={categoryId}
                category={{
                  categoryId,
                  categoryName,
                  categoryDescription,
                  products,
                }}
              />
            );
          }
        )}```

props로 받을 때
```
const Category = ({
  category: { categoryId, categoryName, categoryDescription, products },
})
```

Reply:
아 네네 props로 보낼 수 있는 방법은 많은데, 
* map 쓸 때는 매개변수자리에서 구조분해 할 수 있는걸 모르는 분들이 계셔서
* HOXY나 해서 말씀드린부분이구
* spread로 보내는방법이나, props로 하나씩 지정해줘서 보내는방법 둘중에 정답은 없지만,
* 어떤게 더 가독성과 유지보수에 좋을지는 써보면서 경험해보시는게 좋겠네요 :-)
* 저는 하나씩 필요한것만 props로 보내는게 상위컴포넌트에서 뭘, 어떤이름으로 보냈는지 비교하면서 보기 좋더라구요